package.json

```diff
@@ -17,28 +15,21 @@
     "url": "https://github.com/f1v/redliner.git"
   },
   "dependencies": {
-    "@mdx-js/react": "^1.5.4",
     "classnames": "^2.2.6",
-    "lodash": "^4.17.11",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-popper": "^1.3.3",
-    "react-popper-tooltip": "^2.8.3"
+    "docz": "2.2.0",
```

Comment:
This should be in `devDependencies`. Probably `node-sass` and `gatsby-plugin-sass` as well.